synapse/config/server.py

```diff
@@ -88,6 +91,19 @@ def read_config(self, config):
             "restrict_public_rooms_to_local_users", False,
         )
 
+        self.default_room_version = config.get(
```

Comment:
can you use a local variable for this rather than abuse the member field for a temporary var?